docker/ftest/src/test/java/org/arquillian/cube/servlet/HelloWorldServletTest.java

```diff
@@ -87,15 +87,6 @@ public void should_get_changes_on_container(@ArquillianResource CubeController c
     }
 
     @Test
-    public void should_copy_files_from_container(@ArquillianResource CubeController cubeController, @ArquillianResource CubeID cubeID) throws IOException {
```

Comment:
Currently ftest is using 1.23 docker remote api. If look at [copyFileFromContainerCmd method](https://github.com/docker-java/docker-java/blob/master/src/main/java/com/github/dockerjava/api/DockerClient.java#L176) which is deprecated from version 1.20. So there is no use of this test for this new version.

this test was using remote docker api 1.12 previously.